### PR TITLE
Make common value for enabling app-mgmt and conn-svc

### PR DESCRIPTION
**Pull request by crabtree**
**Description**

Changes proposed in this pull request:

- Make common value for enabling app-mgmt and conn-svc

**Related issue(s)**

### CODE REVIEW

File: resources/application-connector/README.md [hunk 1]
```diff
@@ -7,9 +7,51 @@ The Application Connector connects an external solution to Kyma.
...
+- Application operator
 - Application registry
+- Connection token handler
 - Connector service
 
-## Installation
+## Connector Service
 
-The Application Connector is a part of the Kyma core and it installs automatically.
+The Connector Service is responsible for generating and sending back client certificates based on Certificate Signing Request (CSR). It supports two modes: standalone and central. In standalone mode, the Connector Service works on and for the cluster in which it is deployed. In the central mode, Connector Service supports multiple clusters. By default, the installation of the Connector Service installation is set to standalone.
```
Comment (tomekpapiernik) [inline resources/application-connector/README.md]:
```suggestion
The Connector Service is responsible for generating and sending back client certificates based on Certificate Signing Request (CSR). It supports two modes: standalone and central. In standalone mode, the Connector Service works on and for the cluster in which it is deployed. In the central mode, Connector Service supports multiple clusters. By default, the Connector Service installs as a standalone component.
```

File: resources/application-connector/README.md [hunk 2]
```diff
@@ -7,9 +7,51 @@ The Application Connector connects an external solution to Kyma.
...
+
+### Install Connector Service as central
```
Comment (tomekpapiernik) [inline resources/application-connector/README.md]:
```suggestion
### Install Connector Service as a standalone component
```

File: resources/application-connector/README.md [hunk 3]
```diff
@@ -7,9 +7,51 @@ The Application Connector connects an external solution to Kyma.
...
+
+To install Connector Service configured in the central mode you must override values as presented in the following example:
```
Comment (tomekpapiernik) [inline resources/application-connector/README.md]:
```suggestion
To install the Connector Service as a standalone component, you must override several values of the Kyma Installer. Run: 
```

File: resources/application-connector/README.md [hunk 4]
```diff
@@ -7,9 +7,51 @@ The Application Connector connects an external solution to Kyma.
...
+  labels:
+    installer: overrides
+    kyma-project.io/installation: ""
+data:
+  connector-service.deployment.args.central: "true"
+  connector-service.tests.central: "true"
+EOF
+```
+
+### Install without Connector Service
```
Comment (tomekpapiernik) [inline resources/application-connector/README.md]:
```suggestion
### Install Application Connector without the Connector Service
```

File: resources/application-connector/README.md [hunk 5]
```diff
@@ -7,9 +7,51 @@ The Application Connector connects an external solution to Kyma.
...
+
+You can install Application Connector without Connector Service. For example in preparing an environment for working with central Connector Service. Installation without Connector Service will also disable the possibility to create Application from the UI. To perform the installation without Connector Service you must override values as presented in the following example:
```
Comment (tomekpapiernik) [inline resources/application-connector/README.md]:
```suggestion
You can install the Application Connector (AC) without the Connector Service. Use this approach when preparing an environment for working with a Connector Service deployed as a standalone component. 
To install the AC without the Connector Service, you must override several values of the Kyma Installer. Run:
>**NOTE:** Installing AC without the Connector Service disables creating new Applications through the Console UI.
```

File: resources/application-connector/README.md (hunk 2 above)
Comment (crabtree) [reply resources/application-connector/README.md]: But this applies to the Connector Service working as the central, not standalone.

File: resources/application-connector/README.md (hunk 3 above)
Comment (janmedrek) [reply resources/application-connector/README.md]: I would suggest slight merge of two versions:
```
To install the Connector Service configured in the central mode, you must override several values of the Kyma Installer. Run: 
```

File: resources/application-connector/README.md (hunk 4 above)
Comment (janmedrek) [reply resources/application-connector/README.md]: Please apply those suggestions

File: resources/application-connector/README.md (hunk 5 above)
Comment (janmedrek) [reply resources/application-connector/README.md]: Same here :)

File: resources/application-connector/README.md (hunk 1 above)
Comment (janmedrek) [reply resources/application-connector/README.md]: This one should also be applied

**Pull request merged**